pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -78,43 +78,45 @@ func LowNodeUtilization(ctx context.Context, client clientset.Interface, strateg
 		return
 	}
 
-	thresholds := strategy.Params.NodeResourceUtilizationThresholds.Thresholds
-	targetThresholds := strategy.Params.NodeResourceUtilizationThresholds.TargetThresholds
-	if err := validateStrategyConfig(thresholds, targetThresholds); err != nil {
+	strategyConfig := strategy.Params.NodeResourceUtilizationThresholds
+	if err := validateStrategyConfig(strategyConfig); err != nil {
```

Comment:
Please put the changes generalizing `thresholds` and `targetThresholds` into `strategyConfig.thresholds` and `strategyConfig.targetThresholds` into a separate commit. It will be easier to follow the changes.